scaffolding-chef-inspec/lib/scaffolding.sh

```diff
@@ -167,6 +159,7 @@ interval = 1800
 splay = 1800
 splay_first_run = 0
 log_level = 'warn'
+report_to_stdout = true
```

Comment:
This line was needed for the latest version of the scaffolding, but it does not seem like that code is checked in yet. I was seeing the below error without this line added. 
```
whale-server-tests.default(O): Failed to load JSON configuration: 765: unexpected token at '{
whale-server-tests.default(O):     "target_id": "334ab3d7b1c9463ab2545891e8d371fb",
whale-server-tests.default(O):     "reporter": {
whale-server-tests.default(O):       "cli": {
whale-server-tests.default(O):         "stdout":
whale-server-tests.default(O):       },
whale-server-tests.default(O):       "json": {
whale-server-tests.default(O):         "file": "/hab/svc/whale-server-tests/logs/inspec_last_run.json"
whale-server-tests.default(O):       },
whale-server-tests.default(O):       "automate" : {
whale-server-tests.default(O):         "url": "http://requestbin.sjcmmsn.com/1chvimq1/data-collector/v0/",
whale-server-tests.default(O):         "token": "none",
whale-server-tests.default(O):         "node_name": "7d0be81be198",
whale-server-tests.default(O):         "verify_ssl": false
whale-server-tests.default(O):       }}
whale-server-tests.default(O):     ,
whale-server-tests.default(O):     "compliance": {
whale-server-tests.default(O):      "server" : "http://requestbin.sjcmmsn.com/1chvimq1",
whale-server-tests.default(O):      "token" : "none",
whale-server-tests.default(O):      "user" : "none",
whale-server-tests.default(O):      "insecure" : true,
whale-server-tests.default(O):      "ent" : "automate"
whale-server-tests.default(O):     }
whale-server-tests.default(O): }
whale-server-tests.default(O): '
whale-server-tests.default(O): Config was: {
whale-server-tests.default(O):     "target_id": "334ab3d7b1c9463ab2545891e8d371fb",
whale-server-tests.default(O):     "reporter": {
whale-server-tests.default(O):       "cli": {
whale-server-tests.default(O):         "stdout":
whale-server-tests.default(O):       },
whale-server-tests.default(O):       "json": {
whale-server-tests.default(O):         "file": "/hab/svc/whale-server-tests/logs/inspec_last_run.json"
whale-server-tests.default(O):       },
whale-server-tests.default(O):       "automate" : {
whale-server-tests.default(O):         "url": "http://requestbin.sjcmmsn.com/1chvimq1/data-collector/v0/",
whale-server-tests.default(O):         "token": "none",
whale-server-tests.default(O):         "node_name": "7d0be81be198",
whale-server-tests.default(O):         "verify_ssl": false
whale-server-tests.default(O):       }}
whale-server-tests.default(O):     ,
whale-server-tests.default(O):     "compliance": {
whale-server-tests.default(O):      "server" : "http://requestbin.sjcmmsn.com/1chvimq1",
whale-server-tests.default(O):      "token" : "none",
whale-server-tests.default(O):      "user" : "none",
whale-server-tests.default(O):      "insecure" : true,
whale-server-tests.default(O):      "ent" : "automate"
whale-server-tests.default(O):     }
whale-server-tests.default(O): }
```

Reply:
Yep! This is fixed in #75 but has not been released yet.